addons/mail/models/mail_thread.py

```diff
@@ -3970,6 +3970,24 @@ def _notify_by_web_push_prepare_payload(self, message, msg_vals=False):
         else:
             icon = '/web/static/img/odoo-icon-192x192.png'
 
+        if not body and not message.body and message.attachment_ids:
```

Comment:
Line 3964 is supposed to already take care of reading message.body is body is not provided